README.md

```diff
@@ -0,0 +1,37 @@
+# Deploy
+
+**THIS IS CURRENTLY WIP, AND PROBABLY DOES NOT WORK AT ALL**
+
+## Idea
+
+`deploy.sh` is a simple script that parses a flake passed to it. It expects the following outputs:
+
+```
+deploy
+|--(optional)sshUser
+|--nodes
+   |--<NODE>
+      |--(optional)sshUser
+      |--hostname # hostname of node
+      |--profiles
+         |--<PROFILE>
+            |--(optional)user # user that owns the profile; will be set to $sshUser if omitted
+            |--(optional)sshUser
+            |--(optional)activate # activation script to run after installing the profile; will be ran as $profileUser; no activation if omitted
+            |--path # path that should be linked to profile
+         |--<PROFILE>...
+   |--<NODE>...
+
+```
+
+This structure is subject to change a lot during initial development.
+
+## Usage
+
+`nix run github:serokell/deploy your-flake#node.profile`
```

Comment:
This "usage" does what exactly? Does it run the script?

Reply:
Yep, `nix run ` will run the script, and all the arguments after `github:serokell/deploy` are passed to it.

Reply:
@mkaito, pls, resolve the convo if this explanation ~of this weird syntax~ sound good to you.